runner/actions-runner-dind-rootless.ubuntu-20.04.dockerfile

```diff
@@ -149,6 +150,14 @@ RUN export ARCH=$(echo ${TARGETPLATFORM} | cut -d / -f2) \
     && which docker-compose \
     && docker compose version
 
+RUN export ARCH=$(echo ${TARGETPLATFORM} | cut -d / -f2) \
+    && if [ "$ARCH" = "aarch64" ]; then export ARCH=arm64 ; fi \
+    && if [ "$ARCH" = "x86_64" ] || [ "$ARCH" = "i386" ]; then export ARCH=amd64 ; fi \
+    && mkdir -p /usr/libexec/docker/cli-plugins \
```

Comment:
It seems like this fails for me like `mkdir: cannot create directory '/usr/libexec/docker': Permission denied`, probably due to the preceding `USER runner` at L136!

Reply:
@mumoshu Sorry for the delay, should be fixed now :)